azurelinuxagent/ga/exthandlers.py

```diff
@@ -2292,6 +2326,59 @@ def is_continue_on_update_failure(self):
     def supports_multiple_extensions(self):
         return self.data['handlerManifest'].get('supportsMultipleExtensions', False)
 
+    def get_resource_limits(self, extension_name, str_version):
+        """
+        Placeholder values for testing and monitoring the monitor extension resource usage.
+        This is not effective after nov 30th.
+        """
+        if ExtHandlerInstance.is_azuremonitorlinuxagent(extension_name):
+            if LooseVersion(str_version) < LooseVersion("1.12"):
+                test_man = {
```

Comment:
Can you make this part of the conf file too? That would make it easier to dynamically test it on other extensions too (like the test extension) in the future without a lot of changes and would keep the agent long tail clean

Reply:
sorry I didn't get it. which part you are suggesting in conf file? is it entire resourcelimits json?